pennylane/ops/cv.py

```diff
@@ -189,10 +189,12 @@ class Displacement(CVOperation):
     Phase space displacement.
 
     .. math::
-       D(\alpha) = \exp(\alpha a^\dagger -\alpha^* a)
+       D(r,\phi) = D(\alpha) = \exp(\alpha a^\dagger -\alpha^* a)
```

Comment:
Since `r` is also used for squeezing gate, and `DisplacedSqueezedState` needed to use `a` to avoid this collision, should we just change variable name from `r` to `a` here?

Reply:
Good idea. implemented in d260c5935253b07141a7555e11b7ffc5bcacf0df.